docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,108 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 mapped addresses in `contains`. ({issue}`18497`)
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
+* Improve performance of `GROUP BY`, `DISTINCT`, and other hashing operations. ({issue}`19059`)
```

Comment:
Thats a big one! Do we need to rephrase to something like

Improve performance of hashing operations used for `GROUP BY`, `DISTINCT, and other aggregations.